docs/excel/custom-functions-web-reqs.md

```diff
@@ -119,36 +119,55 @@ ws.onerror(error){
 }
 ```
 
-## Streaming functions
+## Streaming and cancelable functions
 
-Streaming custom functions enable you to output data to cells repeatedly over time, without requiring a user to explicitly request data refresh. The following code sample is a custom function that adds a number to the result every second. Note the following about this code:
+Streaming custom functions enable you to output data to cells repeatedly over time, without requiring a user to explicitly request data refresh.
```

Comment:
Consider: "user to explicitly request data refresh" -> "user to explicitly refresh anything."